docs/integration-services/connection-manager/ole-db-connection-manager.md

```diff
@@ -81,6 +81,49 @@ manager: craigg
+
+To use managed identity authentication, follow these steps:
+
+1. **Create a group in Azure AD.** Make the managed identity a member of the group.
+    
+   1. Find the data factory managed identity from the Azure portal. Go to your data factory's **Properties**. Copy the SERVICE IDENTITY ID.
+    
+   1. Install the [Azure AD PowerShell](https://docs.microsoft.com/powershell/azure/active-directory/install-adv2) module. Sign in by using the `Connect-AzureAD` command. Run the following commands to create a group and add the managed identity as a member.
+      ```powershell
+      $Group = New-AzureADGroup -DisplayName "<your group name>" -MailEnabled $false -SecurityEnabled $true -MailNickName "NotSet"
+      Add-AzureAdGroupMember -ObjectId $Group.ObjectId -RefObjectId "<your data factory managed identity object ID>"
+      ```
+    
+1. **[Provision an Azure Active Directory administrator](https://docs.microsoft.com/en-us/azure/sql-database/sql-database-aad-authentication-configure#provision-an-azure-active-directory-administrator-for-your-azure-sql-database-server)** for your Azure SQL server on the Azure portal if you haven't already done so. The Azure AD administrator can be an Azure AD user or Azure AD group. If you grant the group with managed identity an admin role, skip steps 3 and 4. The administrator will have full access to the database.
+
+1. **[Create contained database users](https://docs.microsoft.com/en-us/azure/sql-database/sql-database-aad-authentication-configure#create-contained-database-users-in-your-database-mapped-to-azure-ad-identities)** for the Azure AD group. Connect to the database from or to which you want to copy data by using tools like SSMS, with an Azure AD identity that has at least ALTER ANY USER permission. Run the following T-SQL: 
+    
+    ```sql
+    CREATE USER [your AAD group name] FROM EXTERNAL PROVIDER;
+    ```
+
+1. **Grant the Azure AD group needed permissions** as you normally do for SQL users and others. For example, run the following code:
+
+    ```sql
+    ALTER ROLE [role name] ADD MEMBER [your AAD group name];
+    ```
+
+1. **Configure managed identity authentication** for the OLE DB connection manager.
+
+   1. Set the connection manager property **ConnectUsingManagedIdentity** to **True**. Please note that all other authentication methods (e.g., integrated security, password) configured on the OLE DB connection manager will be **overrided** when managed identity authentication is used to establish database connection.
+   1. In the connection manager property **ConnectionString**, set the connection property **Provider** to **MSOLEDBSQL**, which refers to the latest [**Microsoft OLE DB Driver for SQL Server**](https://go.microsoft.com/fwlink/?linkid=871294). To use managed identity authentication with OLE DB connection manager, this new provider must be used in your OLE DB connection manager.

```

Comment:
There is also the UI way to choose the new provider in addition to setting the connection string directly.